### PR TITLE
Correct `cmd` when `exectuable-find curl`

### PR DESCRIPTION
**Pull request by duckwork**
I've made it match the wget part a little before it.  I'm not sure if that's correct.

### CODE REVIEW

**Comment by basil-conto** [comment]
See also https://github.com/Boruch-Baum/emacs-crossword/issues/3#issuecomment-762934248.

File: crossword.el [hunk 1]
```diff
@@ -1568,7 +1568,9 @@ TITLE is a string. PATH is the destination."
              cmd     (list "wget" "-P" path url)))
      ((executable-find "curl")
        (setq cmd-str (format "curl -o %s%s %s" path (file-name-nondirectory url) url)
-             cmd     (list "wget" "-P" path url)))
+             cmd     (list "curl" "-o" (concat path
+                                               (file-as-nondirectory url))
```
Comment (basil-conto) [inline crossword.el]: `file-as-nondirectory` -> `file-name-nondirectory`

Comment (duckwork) [reply crossword.el]: Thanks for the catch!

**Comment by Boruch-Baum** [comment]
> I've made it match the wget part a little before it. I'm not sure if that's correct.

Thank you, and to Basil, whose original meaning I didn't understand.

**Comment by Boruch-Baum** [comment]
It turned out to need a matching close parens, corrected in 366ec45.
Tested and works.

**Pull request merged**